custom_components/tuya_local/devices/ustellar_ut99911_rgb.yaml

```diff
@@ -0,0 +1,174 @@
+name: Smart Monitor Light Bar
+products:
+  - id: oqtk22eg6jxorl78
+    name: Ustellar UT99911-RGB
+primary_entity:
+  entity: light
+  name: Front LED (White)
```

Comment:
I don't recommend brackets in names, they may complicate the entity ids.  If you need the info, then "Front white LED" and "Rear RGB LED" would be better.

Reply:
Renamed, thank you.